### ISSUE TITLE
The string "config" is being parsed into the config object

### ISSUE THREAD

**Issue by amitkeret** [Nerwyn/service-call-tile-feature#28]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
3.3.3

**Browser and Operating System:**
Firefox 126, Windows 11

**Description of problem:**
`config` is being parsed into the config object, where it shouldn't
When assigning a string value to something with `config` in it, it gets parsed.

**Your Full Tile Config**
```yaml
type: tile
entity: sensor.ha_add_ons_updates
icon: mdi:package-variant
features:
  - type: custom:service-call
    entries:
      - type: button
        tap_action:
          action: navigate
          # The following line contains "config" and is getting parsed.
          navigation_path: /config/updates
        label: |
          {% if states(config.entity) > 0 %}
            {{ VALUE }} pending
          {% else %}
            Up-to-date
          {% endif %}
```

**Additional information:**
I'm using this button in a view with the url `/dashboard/admin`
Trying to click the button above just takes me back to `/dashboard/0`.
If you change `navigation_path` to `config/updates` (no leading slash), you will navigate to `/dashboard/admin/[object Object]/updates`

**Comment by Nerwyn** [comment]
Ah yikes, that's due to some legacy string interpolation logic I kept to not break old configs. I'm out of town but once I get to my machine I'll limit string interpolation to certain fields.

**Comment by Nerwyn** [comment]
Fixed in [3.3.6](https://github.com/Nerwyn/service-call-tile-feature/releases/tag/3.3.6).
